config.json

```diff
@@ -18,6 +18,17 @@
 	    ]
 	},
 	"propagatedmount": "/mnt",
+	"mounts": [
+		{
+		"name": "KeyPath",
+		"description": "Path to SSH keys",
+		"source": "/tmp",
+		"destination": "/tmp/sshfs",
```

Comment:
Don't we need stronger isolation than that? Can we not run multiple instances of a plugin with different configuration?

> This is because changing the setting to accept a different path requires stopping any containers that are using the volume created by the plugin, and removing those volumes.

I see. Is there a way to parameterize it each mount?

It seems very dangerous to let each mount have access to all the remotes.

cc @vieux